packages/gatsby/src/utils/webpack.config.js

```diff
@@ -227,6 +227,7 @@ module.exports = async (
             })
         ),
         "global.hasPartialHydration": isPartialHydrationEnabled,
+        "_CFLAGS_.GATSBY_MAJOR": JSON.stringify(_CFLAGS_.GATSBY_MAJOR),
```

Comment:
Why do we need that?

Reply:
Done in [`3055b95` (#36489)](https://github.com/gatsbyjs/gatsby/pull/36489/commits/3055b9567adb672173aeab99a3b83ed21753cc78), but this is optimistic change that I will be veryfying now